ci/terraform/oidc/dynamo-policies.tf

```diff
@@ -957,3 +985,19 @@ resource "aws_iam_policy" "dynamo_auth_session_delete_policy" {
 
   policy = data.aws_iam_policy_document.dynamo_auth_session_delete_policy_document.json
 }
+
+resource "aws_iam_policy" "dynamo_orch_session_logout_policy" {
+  name_prefix = "dynamo-orch-session-logout-policy"
+  path        = "/${var.environment}/oidc-shared/"
+  description = "IAM policy for managing delete and describe permissions to the orch session table"
+
+  policy = data.aws_iam_policy_document.dynamo_orch_session_logout_policy_document.json
+}
+
```

Comment:
Oh the other comment I got halfway though writing earlier - can we label these policies by what permissions they give, rather than what they're used for?